lib/eventsource-polyfill.js

```diff
@@ -1,5 +1,6 @@
 var EventSource = require('./eventsource')
 if (typeof window === 'object') {
   window.EventSource = window.EventSource || EventSource
+} else if (typeof global === 'object') {
+  global.EventSource = global.EventSource || EventSource
```

Comment:
hmmm… I'd say neither. In node, we'd probably just want to set `module.exports`. So, `module.exports = EventSource`. Take a look at the UMD spec: https://github.com/umdjs/umd